src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -83,17 +54,12 @@ interface EventDefinition {
 type FlowNode = TFlowNode | TActivity | TReceiveTask | TEventBasedGateway | TTextAnnotation;
 
 export default class ProcessConverter {
-  deserialize(processes: string | TProcess | (string | TProcess)[]): void {
-    try {
-      convertedFlowNodeBpmnElements.clear();
```

Comment:
We have to clean the maps before each new parsing, because it's keep the old converted elements of the previous file.
Some elements can be in conflict if there are the same id.

Reply:
Forgot my comment. I didn't see that we instantiate a new BPMNParser (and new ConvertedElements) at each loading ^^